src/Entity/Categories.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\CategoriesRepository")
+ */
+class Categories
+{
+    /**
+     * @ORM\Id
+     * @ORM\GeneratedValue
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $categoryName;
+
+    /**
+     * @ORM\Column(type="string")
+     */
+    private $categoryShortName;
+
+    /**
+     * @return mixed
+     */
+    public function getId(): int
+    {
+        return $this->id;
+    }
+
+    /**
+     * @param int $id
+     * @return bool
+     */
+    public function setId(int $id): bool
+    {
+        $this->id = $id;
+        return true;
+    }
+
+    /**
+     * @return string
+     */
+    public function getCategoryName(): string
+    {
+        return $this->categoryName;
+    }
+
+    /**
+     * @param String $categoryName
+     * @return bool
+     */
+    public function setCategoryName(String $categoryName): bool
+    {
+        $this->categoryName = $categoryName;
+        return true;
```

Comment:
po co zwracać tego boola? Szczególnie, że i tak nie ma możliwości zwrócenia niczego innego poza true?

Reply:
Wtedy nie zwroce nic, a o ile sie orientuje zawsze trzeba cos zwracac.

Reply:
Zwracasz void - czyli nic, ale to też jest poprawne